ModuleFourTasks/Task1/Program.cs

```diff
@@ -7,7 +7,19 @@ internal class Program
     {
         private static void Main(string[] args)
         {
+            const string finishWords = @"Click any button to close application.";
+            if (args.Length < 1)
+            {
+                Console.WriteLine("No arguments were provided. Try again...");
+            }
+            else
+            {
+                args.ToList().ForEach((item) => Console.Write(item[0]));
```

Comment:
The program should handle user input and show the first character of **each row entered**. Use `Console.Read` or `Console.ReadKey` for that.

Reply:
Fixed in e6b279e.